src/Components/SocialIconsEmailPlacement.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from 'react-router-dom'
+import '../Styling/SocialIconsEmailPlacement.css';
+
+function SocialIconsEmailPlacement(){
+    return(
```

Comment:
When calling this component, all 3 icons + “social_icon_line” will be displayed and I dont think that’s what we want when creating the sidebars...There’s a couple things to consider with your current implementation...Let’s discuss next time we meet.